src/components/NavLinks/SideBarLinks.js

```diff
@@ -85,8 +89,9 @@ const SelectedListItem = ({ items, iconImage }) => {
 };
 
 SelectedListItem.propTypes = {
-  items: PropTypes.arrayOf(PropTypes.array),
-  iconImage: PropTypes.arrayOf(PropTypes.string)
+  items: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
```

Comment:
can we conver this to something more readable? I would think it's better to have an array of objects then just accest the different properties of the object insted of position 0 and 1